routes/newsletter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+const newsletters = require("../controllers/newsletter");
+const { validateEmail } = require("../middlewares/validators/newsletter");
+
+router.post("/subscribe", validateEmail, newsletters.addEmail);
+
+router.get("/", newsletters.getMailingList);
+
+router.post("/unsubscribe", validateEmail, newsletters.removeEmail);
```

Comment:
Shouldn't we remove this as discussed? 
Currently, if we add this then I (or any user) can disable newsletter for any email.

Reply:
I don't remember being discussed where we are removing unsubscribe feature. we discussed that it is necessary as if user wants to discontinue then he will just report spam to RDS mailing id if he/she does not see option of unsubscribe.
@pallabez to address your question, we are not going to give an unsubscribe option on website, it will be inside the newsletter mails only and when user clicks that link, his/her id will be shared as per the account where newsletter was sent, so NO other person can unsubscribe the newsletter for you. @rohan09-raj  pls verify once if that is not the case or I missed some discussion

Reply:
>  his/her id will be shared as per the account where newsletter was sent.
So we are sending the newsletter document Id?